### PR TITLE
Change fixed width header to reduced width header

### PR DESCRIPTION
**Pull request by BobbySig**
Prevents header controls from being cut off when the fixed width header option is enabled, but no longer truly fixes the width. This only reduces the max-width to 925px, which does the same thing until the window is less than the width of the header. See @Wolvan 's concerns in the discussion for #785 for more background and a better explanation of the original problem.

Also mentioning @nightpool because they asked about this.

### CODE REVIEW

File: Extensions/classic_header.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 //* TITLE Header Options **//
-//* VERSION 2.3.1 **//
+//* VERSION 2.3.2 **//
 //* DESCRIPTION Customize the header. **//
 //* DEVELOPER STUDIOXENIX **//
```
Comment (Wolvan) [inline Extensions/classic_header.js]: Mind changing this to new-xkit?

**Comment by Wolvan** [comment]
Apart from the developer change this LGTM

File: Extensions/classic_header.js (hunk 1 above)
Comment (BobbySig) [inline Extensions/classic_header.js]: Done. Committing now.

**Comment by Wolvan** [comment]
LGTM @homu r+

**Pull request merged**